v2/datastream-to-spanner/terraform/samples/mysql-sharded-end-to-end/transformationContext.json

```diff
@@ -0,0 +1,6 @@
+{
+  "SchemaToShardId": {
```

Comment:
This seems to be a new construct

Reply:
No this is the same as the `transformationContext.json` generated by SMT today, I have not changed anything in the Dataflow job.